infracost-usage-example.yml

```diff
@@ -11,13 +11,13 @@ resource_usage:
     monthly_requests: 20000 # Monthly private certificate requests.
```

Comment:
can you remove the reformatting of this file - so the usage that you haven't remain unchanged.